pkg/service/backup/service_backup_restore_integration_test.go

```diff
@@ -335,21 +335,40 @@ func TestRestoreTablesSmokeIntegration(t *testing.T) {
 		testParallel  = 3
 	)
 
-	target := RestoreTarget{
-		Location: []Location{
-			{
-				DC:       "dc1",
-				Provider: S3,
-				Path:     testBucket,
-			},
+	location := []Location{
+		{
+			DC:       "dc1",
+			Provider: S3,
+			Path:     testBucket,
 		},
-		Keyspace:      []string{testKeyspace},
-		BatchSize:     testBatchSize,
-		Parallel:      testParallel,
-		RestoreTables: true,
 	}
-
-	smokeRestore(t, target, testKeyspace, testLoadCnt, testLoadSize)
+	locStr, err := json.Marshal(location)
+	if err != nil {
+		t.Fatal(err)
+	}
+	tablesTargetProps := fmt.Sprintf(`
+{
+  "location": %s,
+  "keyspace": [
+    "%s"
+  ],
+  "snapshot_tag": "sm_20060102150405UTC",
+  "batch_size": %d,
+  "parallel": %d,
+  "restore_tables": true
+}`, locStr, testKeyspace, testBatchSize, testParallel)
```

Comment:
I really like the idea to store restore parameters as json, so that we can properly use `GetRestoreTarget` in the tests!

But it could be the case for more tests than just the `testRestoreTablesSmokeIntegration`! Moreover, it does not feel good that we have to hardcode some random snapshot_tag just for the `GetRestoreTarget` to pass and than swap it with an actual one. 
For those reasons, maybe we could write a function, which could take restore task params and convert them to a json accepted by the `GetRestoreTarget`. Maybe this function could accept `RestoreTarget` for simplicity? Then, this function should be used in all restore tests which would also make them more robust.

Reply:
Sure, I can remove snapshot_tag from the payload definition.
But I'm not gonna refactor other tests now.

Please propose follow up PR if you see it as something really helpfull and needed.

Reply:
UPDATE: I cannot remove snapshot_tag.
This field cannot be empty https://github.com/scylladb/scylla-manager/blob/803d2919ccbca134cfe9fc3437cd88a642abd9a1/pkg/service/backup/restore_model.go#L23

Test fails with "not a Scylla Manager snapshot tag, expected format is sm_20060102150405UTC" on GetRestoreTarget() call if the field is empty.